cartography/data/ontology/devices.json

```diff
@@ -0,0 +1,93 @@
+{
+
+    "bigfix": {
+        "nodes": {
+            "BigFixComputer": {
+                "hostname": "computername",
+                "os": "os"
+            }
+        }
+    },
+    "crowdstrike": {
+        "nodes": {
+            "CrowdstrikeHost": {
+                "hostname": "hostname",
+                "os_version": "os_version",
+                "platform": "platform_name",
+                "model": "model",
+                "serial_number": "serial_number",
+                "instance_id": "instance_id"
+            }
+        }
+    },
+    "duo": {
+        "nodes": {
+            "DuoEndpoint": {
+                "hostname": "device_name",
+                "os": "os_family",
+                "os_version": "os_version",
+                "model": "model"
+            },
+            "DuoPhone": {
+                "hostname": "name",
+                "os_version": "os_version",
+                "model": "model",
+                "platform": "platform"
+            }
+        },
+        "rels": [
+            {
```

Comment:
[thinking] I wonder if this could be represented as a list of RelSchema or MatchLinks.

If RelSchema, that won't work because some of the patterns can be complicated.

If Matchlink, that also won't be nice because Matchlinks are more imperative and not declarative.

In this case I think raw cypher (which _is_ declarative) is actually a good fit.